contracts/Peeranha.sol

```diff
@@ -667,19 +671,23 @@ contract Peeranha is IPeeranha, Initializable {
         return PostLib.getVotedUsers(posts, postId, replyId, commentId);
     }
 
-    /*function addUserRating(address userAddr, int32 rating, uint32 communityId) external {
+    function addUserRating(address userAddr, int32 rating, uint32 communityId) external {
```

Comment:
This function must be commented by default.
Let's add comment that it is used for testing only.